hazelcast/src/main/java/com/hazelcast/map/impl/operation/DeleteOperation.java

```diff
@@ -74,20 +73,12 @@ public int getId() {
     @Override
     protected void writeInternal(ObjectDataOutput out) throws IOException {
         super.writeInternal(out);
-
-        // RU_COMPAT_3_10
```

Comment:
Can you remove the `Versioned` interface from this and other classes?

Reply:
done